docs/proposals/label-associated-quota.md

```diff
@@ -0,0 +1,72 @@
+# Label Constrained Quota
+
+## Problem
+Cluster-administrators want to allot a total quota cap to a particular entity (users to start with) and allow that entity to allocate the quota amongst "his" projects as he sees fit.
+Our mechanism for relating entities is label selection, so we can create a `ResourceQuotaAllocation` object at the cluster scope that applies allotment totals against `ResourceQuota`
+objects.  By associating projects to owners using structured labels, it becomes possible to allocate quota the projects "owned" by a given user.
+
+
+## Types
+```go
+type ResourceQuotaAllocation struct{
```

Comment:
Let's imagine that I want to run with an `object-counts` quota and a `compute-resources` quota and a `best-effort` quota for an organization as seen here but with bigger values...

https://github.com/openshift/openshift-docs/pull/1747

As we look to expand on this doc, can we add an example that illustrates how the quota is subdivided?